core/spec/models/spree/variant_spec.rb

```diff
@@ -270,6 +270,58 @@
 
   end
 
+  describe 'exchange_name' do
+    let!(:variant) { create(:variant, option_values: []) }
+    let!(:master) { create(:master_variant) }
+
+    before do
+      variant.option_values << create(:option_value, {
+                                                     name: 'Foo',
+                                                     presentation: 'Foo',
+                                                     option_type: create(:option_type, position: 2, name: 'Foo Type', presentation: 'Foo Type')
+                                                   })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.